src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            System.out.println(lottery.getRandomBall());
+        }
         // create three balls using class Lottery and print information about them in console
```

Comment:
we can remove this comment now since we have already done what was described here

Reply:
fixed. Please check